code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2611,3 +2611,41 @@
 		var/color
 		CONVERT_PH_TO_COLOR(exposed_atom.reagents.ph, color)
 		exposed_atom.add_atom_colour(color, WASHABLE_COLOUR_PRIORITY)
+
+/datum/reagent/ants
+	name = "Ants"
+	description = "A sample of a lost breed of Space Ants (formicidae bastardium tyrannus), they are well-known for ravaging the living shit out of pretty much anything."
+	reagent_state = SOLID
+	color = "#993333"
+	taste_mult = 1.3
+	taste_description = "tiny legs scuttling down the back of your throat."
+	metabolization_rate = 5 * REAGENTS_METABOLISM //1u per second
+	glass_name = "glass of ants"
+	glass_desc = "Bottoms up...?"
+	ph = 4.6 //Ants contain Formic Acid, meaning they would likely be very acidic themselves if you used them as a reagent
+	var/ant_damage = 0 //How much damage the ants are going to be doing (rises with each tick the ants are in someone's body)
+	var/amount_left = 0 //Tells the debuff how many ants we are being covered with.
+
+/datum/reagent/ants/on_mob_life(mob/living/carbon/victim)
+	victim.adjustBruteLoss(max(0.1, round((ant_damage * 0.005),0.1))) //Scales with time. Around 12.5 brute for 50 seconds.
+	if(prob(5)) //Due to the fact this has a chance of happening every cycle, it's more likely to happen than it looks
+		if(prob(2)) //Super rare statement
+			victim.say("AUGH NO NOT THE ANTS! NOT THE ANTS! AAAAUUGH THEY'RE IN MY EYES! MY EYES! AUUGH!!", forced = /datum/reagent/ants)
+		else
+			victim.say(pick("THEY'RE UNDER MY SKIN!!", "GET THEM OUT OF ME!!"), forced = /datum/reagent/ants)
+	if(prob(15))
+		victim.emote("scream")
+	ant_damage += 1
+	..()
+
+/datum/reagent/ants/on_mob_end_metabolize(mob/living/livin)
```

Comment:
it doesn't technically matter since you're not using the argument, but `mob/living/livin` is a bit of a weird var name for an argument.